README.md

```diff
@@ -24,10 +23,19 @@ A simple example extracted from the unit tests:
     jb.destroy();
     assertEquals(ibb, abb);
 ```
-Blosc shared library should be in PATH on Windows or in LD_LIBRARY_PATH on Linux/Unix.
+## Installation
+First of all, you need to install the Blosc library (visit https://github.com/Blosc/c-blosc for more details), in short, if you already have CMake, executing the following commands should do the work:
+```bash
+git clone https://github.com/Blosc/c-blosc.git
+cd c-blosc
+mkdir build
+cd build
+cmake -DDEACTIVATE_ZLIB=ON -DCMAKE_GENERATOR_PLATFORM=x64 ..
```

Comment:
You should not need to `-DDEACTIVATE_ZIB=ON`.  If Windows have issues with zlib, then I'd say that `PREFER_EXTERNAL_ZLIB=OFF` would be a safer bet.